generator/package.json

```diff
@@ -54,10 +53,6 @@
     "format-code": "tsfmt -r",
     "tslint-fix": "tslint --fix --project .",
     "tslint": "tslint --project .",
-    "exe": "yarn run exe:alpine && yarn run exe:mac && yarn run exe:linux",
```

Comment:
How come these were removed?

Reply:
hello, it comes from the commit

```
feat(nexe): remove nexe from the generator as now it's built from the theia repository
```

as it seems that now we don't need anymore the native binary as the generator is already provided in che-theia repository